### ISSUE TITLE
Glow ink on signs breaks emoji rendering

### ISSUE THREAD

**Issue by raqbit** [InnovativeOnlineIndustries/Emojiful#35]
Applying glow ink to a sign will break emoji rendering on it. This issue does not appear when you dye the sign text.

---

**Before applying glow ink**

![image](https://user-images.githubusercontent.com/11144180/147473619-1cca77b7-d1a1-40eb-b98e-3599b9b2e5bd.png)

**After applying glow ink**

![image](https://user-images.githubusercontent.com/11144180/147473666-4e155796-ef9f-4809-b55f-12baaf4c8c35.png)

**Comment by BunkWire2X8** [comment]
Yeah, same here. The emojis are visible in the text editor, but not so much on the sign itself.
